kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/BootstrapServersAnnotation.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.kubernetes.operator;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class BootstrapServersAnnotation {
```

Comment:
Javadoc, something like
```suggestion
/**
 * Represents the value of the kroxylicious.io/bootstrap-servers annotation that's applied to ...
 */
public class BootstrapServersAnnotation {
```